File: DetailViewTabCount/Module/VB/DetailViewTabCountVB.Module/BusinessObjects/Employee.vb
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Xpo

<DefaultClassOptions>
Public Class Employee
	Inherits Person

	Public Sub New(ByVal session As Session)
		MyBase.New(session)
	End Sub

	<Association("Employee-Tasks")>
	Public ReadOnly Property Tasks() As XPCollection(Of Task)
		Get
			Return GetCollection(Of Task)(NameOf(Tasks))
		End Get
	End Property
End Class

File: DetailViewTabCount/Module/CS/DetailViewTabCount.Module/BusinessObjects/Employee.cs
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Xpo;

namespace DetailViewTabCount.Module.BusinessObjects
{
    [DefaultClassOptions]
    public class Employee : Person
    {
        public Employee(Session session) : base(session) { }

        [Association("Employee-Tasks")]
        public XPCollection<Task> Tasks
        {
            get { return GetCollection<Task>(nameof(Tasks)); }
        }
    }
}
